app/templates/client/app/views/forgot-password.html

```diff
@@ -0,0 +1,53 @@
+<div id="dashboard-forgot-password">
+  <div class="page-forgot-password">
+    <div class="forgot-password-header">
+      <section class="logo text-center">
+        <h1>Skeleton</h1>
+      </section>
+    </div>
+
+    <div class="forgot-password-body">
+      <div class="container">
+
+        <div class="alert alert-success fade in" ng-show="passwordResetSent">
+          <i class="fa fa-check-circle fa-fw fa-lg"></i>
+          If the username is registered in our system we'll send you an email with instructions.
+        </div>
+
+        <div class="alert alert-danger fade in" ng-show="ErrorMsg">
+          <i class="fa fa-times-circle fa-fw fa-lg"></i>
+          Request failed.
```

Comment:
improve, there's request fail, and there's `can't login due to too many attempts`